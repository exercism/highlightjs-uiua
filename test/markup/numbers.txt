5 6e3 0 3.2 3/4 ¯2/¯4 ¯1.1 η π τ ∞