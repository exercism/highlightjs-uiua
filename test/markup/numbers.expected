<span class="hljs-number">5</span> <span class="hljs-number">6e3</span> <span class="hljs-number">0</span> <span class="hljs-number">3.2</span> <span class="hljs-number">3/4</span> <span class="hljs-number">¯2</span>/<span class="hljs-number">¯4</span> <span class="hljs-number">¯1.1</span> <span class="hljs-number">η</span> <span class="hljs-number">π</span> <span class="hljs-number">τ</span> <span class="hljs-number">∞</span>